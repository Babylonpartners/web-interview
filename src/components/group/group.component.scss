.group {
  margin: 36rem 0;
}
