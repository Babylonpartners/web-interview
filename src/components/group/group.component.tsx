import React from 'react'

import './group.component.scss'

export const Group = props => {
  return <div className="group">{props.children}</div>
}
