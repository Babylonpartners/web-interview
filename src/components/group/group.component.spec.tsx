import React from 'react'
import renderer from 'react-test-renderer'

import { Group } from './group.component'

describe('Group', () => {
  it('should match snapshot', () => {
    const component = renderer.create(
      <Group>
        <span className="whatever">Some text, yeh!</span>
        <button className="whatever__button">big CTA</button>
      </Group>
    )

    expect(component.toJSON()).toMatchSnapshot()
  })
})
