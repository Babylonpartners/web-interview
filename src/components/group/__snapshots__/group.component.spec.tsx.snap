// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Group should match snapshot 1`] = `
<div
  className="group"
>
  <span
    className="whatever"
  >
    Some text, yeh!
  </span>
  <button
    className="whatever__button"
  >
    big CTA
  </button>
</div>
`;
